yarn.lock

```diff
@@ -19525,6 +20237,109 @@ testcafe@^1.20.1:
     typescript "^3.3.3"
     unquote "^1.1.1"
 
+testcafe@^2.6.2:
```

Comment:
I recall we updated testcafe to support the VRT module, is this worth keeping? I cant remember if there were problems with the updated version or not.